UPGRADE_GUIDE.md

```diff
@@ -5,6 +5,107 @@ To see discussion around these API changes, please refer to the
 [changelog](/CHANGELOG.md) and visit the commits and issues they
 reference.
 
+0.13.x -> 1.0.0
+----------------
+
+- React `0.14.x` is now required
```

Comment:
Hmm, I had all sorts of issues with `context` in 0.13. Migrating to 0.14 made the errors go away.